### ISSUE TITLE
Cannot override the `space-in-parens` rule

### ISSUE THREAD

**Issue by AvraamMavridis** [eslint/eslint#4623]
I am using the airbnb eslint rules, but I can not override the `space-in-parens`  rule, any idea? 

This is my file

```js
module.exports = {
    "extends": "airbnb",
    "rules":{
        "space-before-function-paren": 2,
        "space-in-parens": [2, "always"],
    },
    "plugins": [
        "react"
    ]
};
``` 

ESLINT version: v1.10.3

**Comment by eslintbot** [comment]
@AvraamMavridis Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
@AvraamMavridis what do you mean by "cannot override"? Please tell us what you're expecting and what actually happened.

**Comment by AvraamMavridis** [comment]
@nzakas  airbnb's rules requires no spaces between a functions's argument, so I wanted to change this behavior using `"space-in-parens": [2, "always"],`. I am not 100% sure what is the problem or if there is one, because switching from a `js` eslint config file to a `json` solved it.

**Comment by nzakas** [comment]
It's good to hear you figured out a solution. Unfortunately, you haven't provided us with enough information to further investigate. Please note what we ask for in the first comment.

If you feel like this is no longer worth investigating, feel free to close this issue.

**Comment by platinumazure** [comment]
My guess is the .JS configuration was actually .json and was therefore executed as a side-effect-free expression statement, rather than correctly exported from the module.

@AvraamMavridis Maybe you could confirm? Did your .js file assign to `module.exports` at all?
